### ISSUE TITLE
Adjust `blt frontend` to upgrade the `uiowa/uids` package 

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#4771]
# Problem
`blt frontend` is not updating UIDS properly when run locally.   We've been using workarounds like `rm -rf ./node_modules` and
`yarn cache clean` to make sure it updates properly and removes the previous version.  

# Proposed solution
Investigate `blt frontend` process and find out why `node_modules/@uiowa/@uids` is either not updating or is still cached after running `blt frontend`.

**Comment by richardbporter** [comment]
I'm not sure this is a bug, at least the way its currently titled, because `blt frontend` isn't supposed to update anything, just install what is in the lock file. I think to update uids, we need to run some flavor of [yarn upgrade](https://classic.yarnpkg.com/lang/en/docs/cli/upgrade/) for the `uiowa/uids` package and commit the lock file changes. It might be that there is some issue with yarn upgrade and UIDS?

**Comment by bspeare** [comment]
Thanks @richardbporter I'll update the title.

**Comment by richardbporter** [comment]
I think I can replicate this issue for #4667 which should upgrade uids from c30bf80 to 3666964 with steps: 

```
rm -rf node_modules/@uiowa
git checkout main
ddev blt frontend
(verify version installed, which is hard to do without some identifying info in uids*)
git checkout secondary-navigation-enhancements
ddev blt frontend
```

*I'm using the addition of this file: https://github.com/uiowa/uids/compare/c30bf80...3666964#diff-273a3b9a9c25e6d4241f7001dfca0614b64ca82753b0cb891b701fe3fb39afbe

Those steps should install a new version of uids using this SHA: https://github.com/uiowa/uiowa/pull/4667/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR1216 but that doesn't seem the happen. 

Kinda sounds like what is described here: https://stackoverflow.com/a/48567151. Might have something to do with the fact that this package is not published on npm.

I tried re-adding uids as a dependency of uids_base using the Git URL suggested and that **did** get the new version installed. I pushed it up for now but we can revert if needed: [52cea3d](https://github.com/uiowa/uiowa/pull/4667/commits/52cea3d2d3e64e4c7d52a8787ed0334ec70820ef)

**Comment by richardbporter** [comment]
I reverted [52cea3d](https://github.com/uiowa/uiowa/pull/4667/commits/52cea3d2d3e64e4c7d52a8787ed0334ec70820ef) for now because I think we'll want to test it more. I don't want it to lead to https://github.com/uiowa/uiowa/issues/3014 again.

**Comment by pyrello** [comment]
To be clear, we do not want `blt frontend` to update our dependencies, only to install the versions that are present in the yarn.lock file. We want `blt frontend` to be a reliable way that we can use in test instructions for people other than those actually doing the work to be able to get their frontend dependencies correctly installed.

**Comment by bspeare** [comment]
There are a lot of `uids` changes that we are introducing in https://github.com/uiowa/uids/commit/3666964eb91737e6db6f944fb34883e760b28212 for https://github.com/uiowa/uiowa/pull/4667, so we might want to use that same pr to pull in the `imagemin` removal https://github.com/uiowa/uids/commit/08ffb06fd884a123631eddd5bd4803cf0e6a47b1.

**Comment by briand44** [comment]
See if we can figure this out without publishing on NPM first.

**Comment by richardbporter** [comment]
I'm playing around with this in #4833 and I'm noticing a delay or inability to delete `node_modules/@uiowa` either from my host or the ddev container. I seem to have to SSH into the container and rm -rf it there and then wait a bit for the change to show on my host. Frustrating.

**Comment by richardbporter** [comment]
We tested this out in https://github.com/uiowa/uiowa/pull/4853.

First we changed how uids is specified in uids_base/package.json as a dependency:

`ddev yarn workspace uids_base add https://github.com/uiowa/uids.git#v3.4.6`

Then we committed the changes and did two things to upgrade:
- Manually update the tag version in uids_base/package.json
- `ddev yarn workspace uids_base upgrade @uiowa/uids`

Specifying the tag without the `v` did **not** work.

I pushed up the changes and @GaryRidgway pulled, having 3.4.6 installed previously, he ran `ddev blt frontend` and it updated to 3.5.0 without having to cacheclean or anything.
